doc.md

```diff
@@ -0,0 +1,59 @@
+# xo-test [![Build Status](https://travis-ci.org/vatesfr/xo-test.png?branch=master)](https://travis-ci.org/vatesfr/xo-test)
+
+> Test client for Xo-Server
+
+<a>
+
+## Organization
+
+Reminder: the commands are composed of a **component** and an *action* (**user**.*create*, **job**.*getAll*, **vm**.*delete*,...). 
+
+- The tests are classified by component as shown below :
+```
+src
+├─ [component1]
+|     ├─ __snapshots__
+|     |     └─ [component1].spec.js.snap 
+|     └─ [component1].spec.js
+├─ [component2]
+¦     └─ [component2].spec.js
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
+
+- In each file `[component].spec.js`, you should have :
+```
+describe("[component]", () => {
+  describe(".[action] :", () => {
+    it("what the test does...", () => {
+      // some tests
+    });
+    it("what the test does...", () => {
+      // some tests
+    });
+  });
+});
+```
+
+- There is a helper `_xoWithTestHelpers.js` to manage the temporary items created for the tests (vm, backup, user...). 
+
+
+## Adding a test
+
+- First, check if the main command `[component].[action]` used by the feature you want to test, is already tested.
```

Comment:
We not test `xo-cli`. We test the functioning of `xo-server` using the api provided by it. Maybe we will only test scenarios instead of testing the api methods.